gammapy/estimators/map/excess.py

```diff
@@ -62,6 +61,45 @@ def convolved_map_dataset_counts_statistics(dataset, kernel, mask, correlate_off
         return CashCountsStatistic(n_on_conv.data, background_conv.data)
 
 
+def get_joint_excess_maps(estimator, datasets):
+    """Computes correlated excess and significance for a set of datasets.
+    The significance computation assumes that the model contains
+    one degree of freedom per valid bin in energy in each dataset.
+
+    Parameters
+    ----------
+    estimator : `~gammapy.estimator.ExcessMapEstimator`
+        Excess Map Estimator
+    dataset : `~gammapy.datasets.Datasets`
+        Datasets
+
+    """
+
+    geom = datasets[0].counts.geom.to_image()
+    for dataset in datasets[1:]:
+        if dataset.counts.geom.to_image() != geom:
+            raise ValueError
+
+    ts_sum = Map.from_geom(geom)
+    npred_excess_sum = Map.from_geom(geom)
+    significance = Map.from_geom(geom)
+
+    dof = 0
+    for dataset in datasets:
+        results = estimator.run(dataset)
```

Comment:
Is it not useful to check that the estimator is an excessMapEstimator? By safety

Reply:
added